core/proto/exploration.proto

```diff
@@ -0,0 +1,603 @@
+syntax = "proto3";
```

Comment:
We should be consolidating our protos in a common repository with Oppia Android since they need to be updated in tandem (similar to https://github.com/oppia/oppia-ml-proto). I'm fine with creating that to get these initial protos checked in.

That being said, @seanlip do you have any thoughts on this?

Reply:
No concerns on my end, but let's have the README for that repo define the update flow so we know that this is going to be forward-compatible long-term. (E.g. what needs to be done if that repo is updated; how do developers bring the changes in automatically; etc.) 

If it's too much work to do that now, then let's just keep the proto in Oppia-web first and export it later once we figure out how to manage multiple repos properly.